tomviz/AlignWidget.cxx

```diff
@@ -47,14 +48,16 @@
 #include <QLineEdit>
 #include <QSpinBox>
 #include <QKeyEvent>
+#include <QTableWidget>
+#include <QTableWidgetItem>
 #include <QButtonGroup>
 
 namespace tomviz
 {
 
-AlignWidget::AlignWidget(DataSource* d, QWidget* p, Qt::WindowFlags f)
-  : QWidget(p, f), timer(new QTimer(this)), frameRate(5),
-    unalignedData(d), alignedData(nullptr)
+AlignWidget::AlignWidget(TranslateAlignOperator *op, QWidget* p)
+  : EditOperatorWidget(p), timer(new QTimer(this)), frameRate(5),
```

Comment:
I think the default frameRate is actually 10 instead of 5.
In line 251, you have timer->start(100) and 100 = 1000/10.